pkg/staticnetworkconfig/backward_compatibility.go

```diff
@@ -5,6 +5,9 @@ import "github.com/openshift/assisted-service/internal/common"
 const MinimalVersionForNmstatectl = "4.14"
 
 func NMStatectlServiceSupported(version, arch string) (bool, error) {
+	if version == "" {
```

Comment:
Can you maybe add some comment about the scenario in which version could be empty (also in the PR description)